tests/ui/err_expect.rs

```diff
@@ -0,0 +1,12 @@
+struct MyTypeNonDebug;
```

Comment:
You can test rustfix on this file by having `// run-rustfix` at the top. (Afterwards you need to run `cargo dev bless`)

```suggestion
// run-rustfix

struct MyTypeNonDebug;
```

Reply:
Oh, right, I totally forgot about that :sweat_smile:

Reply:
I always have to look up what the comment needs to say exactly. Working on Clippy and reviewing code has a lot to do with knowing that there is a thing somewhere and then finding it ^^